interface/forms/newpatient/common.php

```diff
@@ -223,6 +223,16 @@ class="css_button link_submit" onClick="top.restoreSession()"><span><?php echo x
                   continue;
               }
 
+              // Fetch acl for category of given encounter. Only if has write auth for a category, then can create an encounter of that category.
+              $postCalendarCategoryACO = fetchPostCalendarCategoryACO($catId);
+              $postCalendarCategoryACO = explode('|',$postCalendarCategoryACO);
+              //$authPostCalendarCategory = acl_check($postCalendarCategoryACO[0], $postCalendarCategoryACO[1]);
+              $authPostCalendarCategoryWrite = acl_check($postCalendarCategoryACO[0], $postCalendarCategoryACO[1],'','write');
```

Comment:
updated code so if no aco is set there is authorization (like in form pernissions if no aco is set for a form in registry)